src/databricks/sqlalchemy/test_local/test_types.py

```diff
@@ -128,6 +128,7 @@ def test_numeric_renders_as_decimal_with_precision_and_scale(self):
     sqlalchemy.types.INT: DatabricksDataType.INT,
     sqlalchemy.types.SMALLINT: DatabricksDataType.SMALLINT,
     sqlalchemy.types.TIMESTAMP: DatabricksDataType.TIMESTAMP,
+    sqlalchemy.types.JSON: DatabricksDataType.STRING,
```

Comment:
Others with more knowledge can chime in, but I wonder if you need to do something similar to TinyIntegerTest to see if the type mapping will actually work. I think this addition makes sure the type will be recognized and translated, but I actually do not know how you would know the value is rendered as a literal string properly.